test/js/specs/happy.js

```diff
@@ -70,151 +71,153 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+        documentUploadConfirmation.verifyCheckReadabilityMessage(documentUploadConfirmationCopy)
+        documentUploadConfirmation.verifyMakeSureDrivingLicenceMessage(documentUploadConfirmationCopy)
+        documentUploadConfirmation.confirmBtn.click()
+        documentUpload.verifyBackOfDrivingLicenceTitle(documentUploadCopy)
+        documentUpload.verifyBackOfDrivingLicenceInstructionMessage(documentUploadCopy)
+        documentUpload.getUploadInput()
+        documentUpload.upload('back_driving_licence.jpg')
+        documentUploadConfirmation.verifyCheckReadabilityMessage(documentUploadConfirmationCopy)
+        documentUploadConfirmation.verifyMakeSureDrivingLicenceMessage(documentUploadConfirmationCopy)
+      })
 
-    it('should return no document message after uploading non-doc image', async () => {
-      goToPassportUploadScreen()
-      uploadFileAndClickConfirmButton('llama.pdf')
-      documentUploadConfirmation.verifyNoDocumentError(documentUploadConfirmationCopy)
-    })
+      it('should upload identity card and verify UI elements', async () => {
+        driver.get(localhostUrl + `?language=${lang}`)
+        welcome.primaryBtn.click()
+        documentSelector.identityCardIcon.click()
+        documentUpload.verifyFrontOfIdentityCardTitle(documentUploadCopy)
+        documentUpload.verifyFrontOfIdentityCardInstructionMessage(documentUploadCopy)
+        uploadFileAndClickConfirmButton('national_identity_card.jpg')
+        documentUpload.verifyBackOfIdentityCardTitle(documentUploadCopy)
+        documentUpload.verifyBackOfIdentityCardInstructionMessage(documentUploadCopy)
+        documentUpload.getUploadInput()
+        documentUpload.upload('back_national_identity_card.jpg')
+        documentUploadConfirmation.verifyCheckReadabilityMessage(documentUploadConfirmationCopy)
+        documentUploadConfirmation.verifyMakeSureIdentityCardMessage(documentUploadConfirmationCopy)
+      })
 
-    it('should upload a document after retrying', async () => {
-      goToPassportUploadScreen()
-      uploadFileAndClickConfirmButton('llama.pdf')
-      documentUploadConfirmation.redoBtn.click()
-      documentUpload.getUploadInput()
-      documentUpload.upload('passport.jpg')
-      documentUploadConfirmation.verifyCheckReadabilityMessage(documentUploadConfirmationCopy)
-    })
+      it('should return no document message after uploading non-doc image', async () => {
+        goToPassportUploadScreen(`?language=${lang}`)
+        uploadFileAndClickConfirmButton('llama.pdf')
+        documentUploadConfirmation.verifyNoDocumentError(documentUploadConfirmationCopy)
+      })
 
-    it('should return file size too large message for doc', async () => {
-      goToPassportUploadScreen()
-      documentUpload.getUploadInput()
-      documentUpload.upload('over_10mb_face.jpg')
-      documentUploadConfirmation.verifyFileSizeTooLargeError(documentUploadConfirmationCopy)
-    })
+      it('should upload a document after retrying', async () => {
```

Comment:
Minor: Think this test description should say "should upload a document on retry"